tests/unit/test_session.py

```diff
@@ -2,13 +2,13 @@
 # Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
 #
 import json
+import logging
 import os
 from typing import Optional
 from unittest import mock
 from unittest.mock import MagicMock
 
 import pytest
-
 import snowflake.snowpark.session
```

Comment:
seems unnecessary. is this added by linter?

Reply:
Reran pre-commit to have consistent formatting.